sdk/OpenTap.Sdk.MSBuild/OpenTap.Sdk.MSBuild.csproj

```diff
@@ -6,6 +6,14 @@
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
     <TargetFramework>netstandard2.0</TargetFramework>
     <PlatformTarget>AnyCPU</PlatformTarget>
+    <!-- We have had this warning for many years and never seen any issues, and I'm tired of seeing it
+        in the build output. It looks something like this:
+
+        warning NU1701: Package 'Microsoft.IO.Redist 6.0.0' was restored
+        using '.NETFramework,Version.....,
+        instead of the project target framework '.NETStandard,Version=v2.0'.
+        This package may not be fully compatible with your project. -->
+    <NoWarn>$(NoWarn);NU1701</NoWarn>
   </PropertyGroup>
```

Comment:
```suggestion
    <!-Quiet a low-grade warning in the style of:
        warning NU1701: Package 'Microsoft.IO.Redist 6.0.0' was restored
        using '.NETFramework,Version.....,
        instead of the project target framework '.NETStandard,Version=v2.0'.
        This package may not be fully compatible with your project. 
        
        This is not seen to be an important issue in our case. 
        -->
    <NoWarn>$(NoWarn);NU1701</NoWarn>
</PropertyGroup>
```